plugins/linux-pipewire/linux-pipewire.c

```diff
@@ -53,5 +53,5 @@ bool obs_module_load(void)
 
 void obs_module_unload(void)
 {
-	obs_pipewire_unload();
+	// This space intentionally left blank.
```

Comment:
Is this comment really necessary? I think we can go without :slightly_smiling_face: